src/workflow_collection.py

```diff
@@ -1,9 +1,17 @@
 from pymongo import MongoClient
 from bson.objectid import ObjectId
-
+from bson.errors import InvalidId
 
 access_url = 'workflow-editor-datastore:27017'
 
+def is_valid_object_id(obj_id):
+    try:
+        wid = ObjectId(obj_id)
```

Comment:
You are here instantiating an ObjectId that you just throw away, and you always instantiate a new one after calling the function. Wouldn't it make more sense to just catch the InvalidId error wherever it may be raised?